### PR TITLE
Added cutting recipes to turn small flowers into dyes

### PR DESCRIPTION
**Pull request by cosmicgelatin**

### CODE REVIEW

**Comment by bageldotjpg** [comment]
This is great, but each recipe is going to need `forge:mod_loaded` conditions for their respective mods

File: src/main/resources/data/abnormals_delight/recipes/atmospheric/cutting/gilia.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  ],
+  "tool": {
+    "tag": "forge:tools/knives"
+  },
+  "result": [
+    {
+      "item": "minecraft:purple_dye",
+      "count": 2
+    }
+  ]
```
Comment (bageldotjpg) [inline src/main/resources/data/abnormals_delight/recipes/atmospheric/cutting/gilia.json:16]: All recipes will need a condition segment below the result, like so:
```  "conditions": [
    {
      "type": "forge:mod_loaded",
      "modid": "neapolitan"
    }
  ]```

Comment (bageldotjpg) [inline src/main/resources/data/abnormals_delight/recipes/atmospheric/cutting/gilia.json:16]: All recipes will need a condition segment below the result, like so:
```
  "conditions": [
    {
      "type": "forge:mod_loaded",
      "modid": "atmospheric"
    }
  ]
```

**Review by bageldotjpg — CHANGES_REQUESTED** [review]
Great work - all that is needed is the proper `forge:mod_loaded` conditions for each mod.

File: src/main/resources/data/abnormals_delight/recipes/atmospheric/cutting/gilia.json (hunk 1 above)
Comment (cosmicgelatin) [reply src/main/resources/data/abnormals_delight/recipes/atmospheric/cutting/gilia.json:22]: Should be good now :thumbsup:

**Review by bageldotjpg — APPROVED** [review]

**Pull request merged**